### PR TITLE
APIのUri変更に対応

### LINKED ISSUE #29: 本の貸し出しの管理
putで本の貸し出しを行う

### PR DESCRIPTION
**Pull request by kkenya**
## 概要
- APIのUri変更に対応する
- ジャンコードを数値に変換
## 技術的変更点
- 文字列をparseIntで数値に変換
## ToDo
- [x] uriの変更
- [x] ジャンコードを数値に変換

### CODE REVIEW

**Comment by kkenya** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@kkenya/BookSeeker-fix-barcodeToNum

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by kkenya** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@kkenya/BookSeeker-fix-barcodeToNum

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by kkenya** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@kkenya/BookSeeker-fix-barcodeToNum

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by kkenya** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@kkenya/BookSeeker-fix-barcodeToNum

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by kkenya** [comment]
マージします

File: networking.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-const baseUri = 'https://go-api-staging.herokuapp.com/books';
+const baseUri = 'https://book-seeker-staging.herokuapp.com/books';
```
Comment (JinOketani) [inline networking.js]: ベースURIだから`https://book-seeker-staging.herokuapp.com`のほうがいいと思う

File: ScanScreen.js [hunk 2]
```diff
@@ -121,7 +121,7 @@ export default class ScanScreen extends React.Component {
             <Text></Text>
           </View>
           <BarCodeScanner style={styles.camera}
-            onBarCodeRead={this._handleBarCodeRead}
+                          onBarCodeRead={this._handleBarCodeRead}
```
Comment (JinOketani) [inline ScanScreen.js]: 謎の空白

File: networking.js (hunk 1 above)
Comment (kkenya) [reply networking.js]: 変更する

File: ScanScreen.js (hunk 2 above)
Comment (kkenya) [reply ScanScreen.js]: スタイルガイドに沿ったスタイリングに変更する

**Pull request merged**